cms/forms/widgets.py

```diff
@@ -78,14 +78,14 @@ def _build_widgets(self):
 
     def _build_script(self, name, value, attrs={}):
         return r"""<script type="text/javascript">
-                var CMS = window.CMS || {};
+                var CMS = window.CMS || {{}};
```

Comment:
???  Is this change OK?

Reply:
@cclauss I don't think so. I think that's being interpreted as a python string when infact it's used by javascript where I don't think that's valid. Though I'm not sure how to get pyupgrade to skip it.

When I implemented the pre-commit config I didn't run it on all files, because that tends to make people nervous when a big change comes in. 

I'll take a look over the rest of these changes.

Reply:
After further review… Actually `pyupgrade` did the right thing and this syntax works as expected.

Reply:
@cclauss @marksweb This can't possibly be correct. This is an invalid syntax and all browsers crash on this. 

Please revert back all the changes that turns `{}` to `{{}}``

<img width="1637" alt="Screenshot 2022-04-02 at 3 29 23 PM" src="https://user-images.githubusercontent.com/537678/161378041-03badb43-f013-4827-bce4-2fb244a9b816.png">

Reply:
Also, I feel pyupgrade shouldn't touch JS strings.